presences/src/main/java/fr/openent/presences/helper/SQLHelper.java

```diff
@@ -0,0 +1,64 @@
+package fr.openent.presences.helper;
+
+import io.vertx.core.json.JsonArray;
+import org.entcore.common.sql.Sql;
+
+import java.util.List;
+
+public class SQLHelper {
+
+    private SQLHelper(){}
+
+
+
+    public static String addLimitOffeset(Integer limit, Integer offset, JsonArray params){
+        String query = "";
+
+        if (offset != null) {
+            query += " OFFSET ?";
+            params.add(offset);
+        }
+
+        if(limit != null){
+            query += " LIMIT ?";
+            params.add(limit);
+        }
+
+        return query;
+    }
+
+    public static String structureId(String structureId, JsonArray params){
+
+        if(structureId != null){
+            params.add(structureId);
+            return " structure_id = ?";
+        }
+        return "";
+    }
+
+    public static String students(List<String> students, JsonArray params){
+        if (!students.isEmpty()) {
+            params.addAll(new JsonArray(students));
+            return " student_id IN " + Sql.listPrepared(students);
+        }
+        return "";
+    }
```

Comment:
```suggestion
    public static String filterList(String field, List<Object> list, JsonArray params){
        if (!list.isEmpty()) {
            params.addAll(new JsonArray(list));
            return String.format(" %s IN %s", field, Sql.listPrepared(list));
        }
        return "";
    }
```
Quitte à être générique, soyons le jusqu'au bout !

dans ton appel ça ferait:
`SqlHelper.filterList('student_id', students, params)`